build/CodeSign.props

```diff
@@ -0,0 +1,274 @@
+<Project>
+
+  <ItemGroup>
+    <!-- Third-party components in Microsoft.AspNetCore.All/App which should be signed.  -->
+    <FilesToSign Include="e_sqlite3.dll" Certificate="$(AssemblySigning3rdPartyCertName)" Container="Microsoft.AspNetCore.All" />
+    <FilesToSign Include="MessagePack.dll" Certificate="$(AssemblySigning3rdPartyCertName)" Container="Microsoft.AspNetCore.All" />
+    <FilesToSign Include="Newtonsoft.Json.Bson.dll" Certificate="$(AssemblySigning3rdPartyCertName)" Container="Microsoft.AspNetCore.All" />
+    <FilesToSign Include="Newtonsoft.Json.dll" Certificate="$(AssemblySigning3rdPartyCertName)" Container="Microsoft.AspNetCore.App" />
+    <FilesToSign Include="Remotion.Linq.dll" Certificate="$(AssemblySigning3rdPartyCertName)" Container="Microsoft.AspNetCore.App" />
+    <FilesToSign Include="SQLitePCLRaw.batteries_green.dll" Certificate="$(AssemblySigning3rdPartyCertName)" Container="Microsoft.AspNetCore.All" />
+    <FilesToSign Include="SQLitePCLRaw.batteries_v2.dll" Certificate="$(AssemblySigning3rdPartyCertName)" Container="Microsoft.AspNetCore.All" />
+    <FilesToSign Include="SQLitePCLRaw.core.dll" Certificate="$(AssemblySigning3rdPartyCertName)" Container="Microsoft.AspNetCore.All" />
+    <FilesToSign Include="SQLitePCLRaw.provider.e_sqlite3.dll" Certificate="$(AssemblySigning3rdPartyCertName)" Container="Microsoft.AspNetCore.All" />
+    <FilesToSign Include="StackExchange.Redis.StrongName.dll" Certificate="$(AssemblySigning3rdPartyCertName)" Container="Microsoft.AspNetCore.All" />
+    <FilesToSign Include="System.Interactive.Async.dll" Certificate="$(AssemblySigning3rdPartyCertName)" Container="Microsoft.AspNetCore.App" />
+
+    <!-- These files should already be signed already by a different leg of the build. They have to be listed again here because we recreate a redistributable which binaries built in other repos. -->
+    <FilesToExcludeFromSigning Include="libuv.dll" />
+    <FilesToExcludeFromSigning Include="Microsoft.AI.DependencyCollector.dll" />
+    <FilesToExcludeFromSigning Include="Microsoft.ApplicationInsights.AspNetCore.dll" />
+    <FilesToExcludeFromSigning Include="Microsoft.ApplicationInsights.dll" />
+    <FilesToExcludeFromSigning Include="Microsoft.Azure.KeyVault.dll" />
+    <FilesToExcludeFromSigning Include="Microsoft.Azure.KeyVault.WebKey.dll" />
+    <FilesToExcludeFromSigning Include="Microsoft.Azure.Services.AppAuthentication.dll" />
+    <FilesToExcludeFromSigning Include="Microsoft.Data.Edm.dll" />
+    <FilesToExcludeFromSigning Include="Microsoft.Data.Edm.resources.dll" />
+    <FilesToExcludeFromSigning Include="Microsoft.Data.OData.dll" />
+    <FilesToExcludeFromSigning Include="Microsoft.Data.OData.resources.dll" />
+    <FilesToExcludeFromSigning Include="Microsoft.Extensions.DiagnosticAdapter.dll" />
```

Comment:
Just a thought: Would it make sense to split out *our* excluded things from Extensions (and wherever else?) into a separate block? Not a big deal I just thought it might be easier to look at, and also easier for later when we need to adjust the list.

Reply:
Will do.